cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/ConversionUtil.java

```diff
@@ -84,4 +85,13 @@ public static boolean xmlContains(InputStream is, String clazz, String ns, Strin
         }
         return false;
     }
+
+    public static Network readCgmesFiles(String dir, String... files) {
+        return readCgmesFiles(new Properties(), dir, files);
+    }
+
+    public static Network readCgmesFiles(Properties properties, String dir, String... files) {
```

Comment:
Maybe consider `readCgmesResources` to avoid confusions with potential reading from file system?

Reply:
Done.